python/api/requirements.txt

```diff
@@ -1,13 +1,33 @@
-# Note this should be synced with the environment.yml so that we can also run ingestion code on the server. 
-
```

Comment:
Its probably due to the top level `requirements.txt` file not being updated.  Updating that now.